lib/wireprotocol/2_6_support.js

```diff
@@ -140,8 +140,10 @@ WireProtocol.prototype.getMore = function(
     var cursorId = typeof r.cursorId === 'number' ? Long.fromNumber(r.cursorId) : r.cursorId;
 
     // Set all the values
-    cursorState.documents = r.documents;
-    cursorState.cursorId = cursorId;
+    cursorState.nextBatches.push({
```

Comment:
Rather then setting `cursorState.documents` directly, we store the results in a queu